backend/app/api/endpoints/groups.py

```
@@ -161,7 +165,7 @@ def list_members(
     group_name: str = Path(
         ..., description="Group name (may contain slashes for subgroups)"
     ),
-    current_user: User = Depends(get_current_user),
+    current_user: User = Depends(get_current_user_jwt_apikey_tasktoken),
```

Ruff (0.15.17)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)